data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/ZkmFilmpalastScraper.java

```diff
@@ -103,7 +100,7 @@ protected Cinema getCinema() {
         WebElement container = bilderEl.findElement(By.xpath(".."));
         WebElement imgEl = container.findElement(By.cssSelector("img"));
 
-        byte imgBytes[];
+        byte[] imgBytes;
```

Comment:
Wel java allows doing it the way you did so thats fine with me, but not with sonar it looks like :D